server.js

```diff
@@ -29,7 +29,7 @@ require("./routes/html-routes.js")(app);
 require("./routes/api-routes.js")(app);
 
 // Syncing our database and logging a message to the user upon success
-db.sequelize.sync().then(() => {
+db.sequelize.sync({ force: true }).then(() => {
```

Comment:
I should remove this.

Reply:
np, I change it on creating a new branch anyways